pkg/store/leveldb/leveldb.go

```diff
@@ -627,6 +637,7 @@ func (s *Store) CreateOrganization(o *bff.Organization) (id string, err error) {
 		return "", err
 	}
 
+	// TODO: this needs to be prefixed by a namespace
```

Comment:
This is kind of a big problem - `trtl` has the concept of namespaces but `leveldb` does not. We've been prefixing the namespace onto the key for the other store methods, but forgot to do it here. This will mean that all the organization UUIDs are not in a contiugous block on disk, making them difficult to iterate over. 

The good news is that leveldb is not used in production, so probably no harm done.

Are there any other leveldb store implementations that need a prefix?

Reply:
story created